docs/data/external-oasis-repos.md

```diff
@@ -1,50 +1,52 @@
+-   RHEL 7.3 (or equivalent) or later.  **This option is recommended**.
+-   RHEL 6 with the aufs kernel module.
 
-Requirements: setting up a CVMFS repository server
---------------------------------------------------
+Additionally,
 
-This document doesn't cover requirements for the GOC computers since those are already set up. The requirements for a CVMFS repository server are that it have installed cvmfs-server and cvmfs client, both version 2.2.2 or greater; version 2.3.3 or greater is recommended on EL6 and required on EL7. This requires using aufs or OverlayFS: aufs requires a modified kernel from CERN for Redhat EL6-based systems, and OverlayFS requires at least EL7.3; the latter uses a standard kernel so it is highly recommended. Also an apache httpd server needs to be enabled. NOTE: multiple cvmfs repositories can be hosted on the same machine.
+-   **User IDs:** If it does not exist already, the installation will create the `cvmfs` Linux user
+-   **Group IDs:** If they do not exist already, the installation will create the Linux groups `cvmfs` and `fuse`
+-   **Network ports:** This page will configure the repository to distribute using Apache HTTPD on port 8000.  At the
+    minimum, the repository needs in-bound access from the OASIS CDN.
+-   **Disk space:** This host will need enough free disk space to host _two_ copies of the software: one compressed
+    and one uncompressed. `/srv/cvmfs` will hold all the published data, while `/var/spool/cvmfs` will contain all
+    the data in the current transaction.
+-   **Root access** will be needed to install.  Software install will be done as an unprivileged user.
+-   **Yum** will need to be [configured to use the OSG repositories](../common/yum).
 
-`/srv/cvmfs` will hold all the published data, so make sure it is large enough to hold all the repositories hosted on the machine. `/var/spool/cvmfs` will hold files during publish, so it should be large enough to hold the most amount of data that might be attempted to be published at once. In addition, on EL7 and cvmfs-server-2.4.1, `/var/spool/cvmfs` needs to be of filesystem type 'ext3' or 'ext4', or the `cvmfs_server` command will not allow it. There is a variable to override the filesystem type check, but other filesystem types are not guaranteed to work (xfs created with `ftype=1` should work with `CVMFS_DONT_CHECK_OVERLAYFS_VERSION=yes`).
+!!! warning
+    RHEL7 only supports Overlay-FS if the underlying filesystem is `ext3` or `ext4`; make sure `/var/spool/cvmfs`
+    is one of these filesystem types.
 
-As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
+    If this is not possible, add `CVMFS_DONT_CHECK_OVERLAYFS_VERSION=yes` to your CVMFS configuration.  It is believed
```

Comment:
Is this something that is not guaranteed to work/not supported/not tested? If so, I think we could use a stronger warning